coap-gateway/service/signUp.go

```diff
@@ -116,6 +116,13 @@ func signUpPostHandler(r *mux.Message, client *Client) {
 		return
 	}
 
+	// client.server.config.APIs.COAP.Authorization.OwnerClaim
+	owner := claim.Owner("sub")
```

Comment:
Why sub?

Reply:
The `client.server.config.APIs.COAP.Authorization.OwnerClaim` path is not valid in v2 yet, once #475 is merged then I can use it.